### PR TITLE
Doc: cost type modified to string

### PR DESCRIPTION
**Pull request by kadhikari**
https://jira.kisio.org/browse/NAVITIAII-2775

### CODE REVIEW

**Review by pbougue — APPROVED** [review]

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -1170,7 +1170,7 @@ direction       | int                    | Angle (in degree) between the previou
 
 |Field|Type|Description|
 |-----|----|-----------|
-|value|float|cost|
+|value|string|cost formatted as string|
```
Comment (mbounabi) [inline documentation/slate/source/includes/apis.md]: Isn't it necessary to mention that it is a 'float' cast into a 'string' ?

Comment (pbougue) [reply documentation/slate/source/includes/apis.md]: We can. The current proposal is ambiguous enough to have integrator check the "real" response before coding :smile: (that's why I approved it).

**Comment by pbougue** [comment]
We also need to decide if we do the last comment on currency in this PR : https://jira.kisio.org/browse/NAVITIAII-2775?focusedCommentId=294427&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-294427
(and if yes : who?)

File: documentation/slate/source/includes/apis.md (hunk 1 above)
Comment (mbounabi) [reply documentation/slate/source/includes/apis.md]: I'm just wondering what will happen if the integrator tries to cast the string in an 'int'

Comment (mbounabi) [reply documentation/slate/source/includes/apis.md]: After discussion: 'float' needs to be specified
```suggestion
|value|string|cost: float formatted as a string|
```

File: documentation/slate/source/includes/apis.md [hunk 2]
```diff
@@ -1170,7 +1170,7 @@ direction       | int                    | Angle (in degree) between the previou
...
 |currency|string|currency|
```
Comment (mbounabi) [inline documentation/slate/source/includes/apis.md]:
```suggestion
|currency|string|currency as specified in input data
```

File: documentation/slate/source/includes/apis.md (hunk 1 above)
Comment (mbounabi) [reply documentation/slate/source/includes/apis.md]:
```suggestion
|value|string|cost: float formatted as string|
```

**Review by pbougue — APPROVED** [review]

**Review by mbounabi — APPROVED** [review]
:1st_place_medal:

**Pull request merged**